packages/hint-classlist-pitfall/src/meta.ts

```diff
@@ -0,0 +1,26 @@
+import { Category } from '@hint/utils-types';
+import { HintScope } from 'hint/dist/src/lib/enums/hint-scope';
+import { HintMetadata } from 'hint/dist/src/lib/types';
+
+import { getMessage } from './i18n.import';
+
+const meta: HintMetadata = {
+    docs: {
+        category: Category.pitfalls,
+        description: getMessage('description', 'en'),
+        name: getMessage('name', 'en')
+    },
+    /* istanbul ignore next */
+    getDescription(language: string) {
+        return getMessage('description', language);
+    },
+    /* istanbul ignore next */
+    getName(language: string) {
+        return getMessage('name', language);
+    },
+    id: 'classlist-add-remove-pitfall',
```

Comment:
The id has to be the same as the folder without `hint-`, in this case `id: 'classlist-pitfall',`.

This is also why the test are failing.